suit_generator/cmd_image.py

```diff
@@ -137,52 +137,13 @@ class ManifestDomain(Enum):
 
 
 class EnvelopeStorage:
-    """Class generating SUIT storage binary in legacy format."""
+    """Base class for generating SUIT storage binary."""
 
     ENVELOPE_SLOT_VERSION = 1
     ENVELOPE_SLOT_VERSION_KEY = 0
     ENVELOPE_SLOT_CLASS_ID_OFFSET_KEY = 1
     ENVELOPE_SLOT_ENVELOPE_BSTR_KEY = 2
 
-    _LAYOUT = [
```

Comment:
I think it would be better to leave it empty, but at least leave it defined.

Reply:
EnvelopeStorage is a parrent class for EnvelopeStorageNrf54h20 class and uses _LAYOUT variable in some methods so it would be good to leve it empty or assign None value.